Tools/autotest/run_in_terminal_window.sh

```diff
@@ -28,6 +28,8 @@ if [ -n "$SITL_RITW_TERMINAL" ]; then
   printf "%q " "$@" >>"$FILEPATH"
   chmod +x "$FILEPATH"
   $SITL_RITW_TERMINAL "$FILEPATH" &
+elif [ -n "$TMUX_PREFIX" ]; then
+  tmux new-window -dn "$name" "$TMUX_PREFIX $*"
 elif [ -n "$TMUX" ]; then
   tmux new-window -dn "$name" "$*"
```

Comment:
Could we put the prefix stuff inside the existing TMUX clause?

It means you do need to set two environment variables but will avoid any cross-product problems.

Reply:
done